src/core/vectorlayercounterpart.js

```diff
@@ -25,6 +23,14 @@ olcs.core.OlLayerPrimitive = function(layerProjection, scene) {
     primitives: primitives
   };
 
-  this.add(billboards);
+  this.rootCollection_.add(billboards);
+  this.rootCollection_.add(primitives);
+};
+
+
+/**
+ * @return {!Cesium.Primitive}
+ */
+olcs.core.VectorLayerCounterpart.prototype.getRootPrimitive = function() {
```

Comment:
I am returning a `Cesium.Primitive` and not a `Cesium.PrimitiveCollection` on purpose and the name was chosen accordingly. I like the `root` which suggests clearly that the hierarchy of Cesium primitives is hidden and should be considered a black box. I could have chosen `getRootCesiumPrimitive` but it sounds redundant.